src/features/DebugSession.ts

```diff
@@ -114,6 +124,87 @@ export class DebugSessionFeature extends LanguageClientConsumer
         ];
     }
 
+    public async createTemporaryIntegratedConsole(session: DebugSession) {
+        const settings = getSettings();
+        this.tempDebugProcess = await this.sessionManager.createDebugSessionProcess(settings);
+        this.tempSessionDetails = await this.tempDebugProcess.start(`DebugSession-${this.sessionCount++}`);
+
+        // NOTE: Dotnet attach debugging is only currently supported if a temporary console is used, otherwise we get lots of lock conflicts from loading the assemblies
```

Comment:
```suggestion
        // NOTE: Dotnet attach debugging is only currently supported if a temporary debug terminal is used, otherwise we get lots of lock conflicts from loading the assemblies.
```